community/samples/README.md

```diff
@@ -1,5 +1,7 @@
 ## Sample apps
 
+Note: Community samples may or may not be maintained by their authors.
```

Comment:
Added active voice and a path towards possible fixes?
```suggestion
Note: Community code samples are contributed and maintained by Knative community members. It is possible that these samples are no longer current and the author might not be actively maintaining their contribution. 
[Learn more about the lifespan of samples](../../contributing/DOCS-CONTRIBUTING.md).  If you find that something is not right, lend a helping hand and fix it in a PR.
```

Reply:
Changes addressed, Thanks for the review!